public/js/utils/source-map.js

```diff
@@ -7,11 +7,16 @@ const { getSource } = require("../selectors");
 const { isEnabled } = require("../feature");
 const { isJavaScript } = require("./source");
 const assert = require("./assert");
+const { prefs } = require("./prefs");
 
 import type { Location } from "./actions/types";
 
 let sourceMapRequests = new Map();
 
+function shouldSourceMap() {
+  return isEnabled("sourceMaps") && prefs.sourceMapsEnabled;
```

Comment:
Sure, that's fine. Honestly I think it might be good if we eventually just use the `Prefs` API itself. It basically works with json files, as far as I know, we can load in whatever json files locally we want. This way in the Firefox panel feature flags are real prefs, and you can toggle them on and off, which is how we've previously been able to with in the panel.

It would be harder to create new flags, yeah, but I think we should also avoid using feature flags for things that should just sit on a branch and be developed a little more too.